lib/companies/companies.ex

```diff
@@ -76,7 +76,13 @@ defmodule Companies.Companies do
   ** (Ecto.NoResultsError)
 
   """
-  def get!(id), do: Repo.get!(Company, id)
+  def get!(id, opts \\ []) do
```

Comment:
TIL, thank you!

Reply:
Nice! @maartenvanvliet great minds think alike! I just implemented this pattern in another project.

It’s also possible, but more work, to work this into a query builder such that you don’t need _N+1_ queries. 

Really like this 👌